web/components/chat/ChatTextField/ChatTextField.tsx

```diff
@@ -167,31 +107,10 @@ export const ChatTextField: FC<ChatTextFieldProps> = ({ defaultText, enabled, fo
         h3: 'p',
       },
     });
-    text.current = sanitized;
-
-    const charCountString = sanitized.replace(/<\/?[^>]+(>|$)/g, '');
```

Comment:
Without this it hits the character limit at four custom emoji, because it's counting the characters in the HTML tags.

Reply:
My thought process was in #1803 I'd be converting the custom emoji into their shortcodes (and the shortcode length is then counted in the message length), I can try changing this PR to count emoji similarly. It wouldn't result in emoji counting as a single character but it's still way less characters than the entire tag.

Reply:
Oh yeah, if you're planning on changing in it with that using shortcodes then no problem!